src/detect-filemagic.c

```diff
@@ -322,13 +322,20 @@ static void *DetectFilemagicThreadInit(void *data)
 
     (void)ConfGet("magic-file", &filename);
     if (filename != NULL) {
-        SCLogInfo("using magic-file %s", filename);
+        if (strncmp(filename,"",strlen(filename)) == 0) {
+            // set filename to NULL on *nix systems so magic_load uses system default path (see man libmagic)
```

Comment:
we use /* */ style comments, even though you'll see the c++ // variant lingering around as well